### PR TITLE
MT : Switch MT committee scraper to use structured HTML rather than PDF  #2098

### PR DESCRIPTION
**Pull request by sriks123**

### CODE REVIEW

**Comment by sriks123** [comment]
@mileswwatkins please review this PR.

**Comment by mileswwatkins** [comment]
@sriks123, can you make edits so that your code passes our Continuous Integration? You can run a Flake 8 test in your code editor, or see what went wrong on Travis CI: https://travis-ci.org/openstates/openstates/builds/341978896

Will check out this PR once it's passing the CI.

**Comment by sriks123** [comment]
@mileswwatkins it has passed the CI test, please review it.

**Comment by sriks123** [comment]
@jamesturk can you review this and tell me what else should I do?

File: openstates/mt/committees.py [hunk 1]
```diff
@@ -3,38 +3,79 @@
...
     def scrape(self, chamber=None, session=None):
         if not session:
             session = max(committee_urls['lower'].keys())
         chambers = [chamber] if chamber else ['upper', 'lower']
 
         for chamber in chambers:
             url = committee_urls[chamber][session]
-            fn, _ = self.urlretrieve(url)
-            yield from self.scrape_committees_pdf(session, chamber, fn, url)
+            if session == '2017':
```
Comment (jamesturk) [inline openstates/mt/committees.py]: this should probably be >= as I think future comms will be HTML too

**Comment by jamesturk** [comment]
when I run this I get

```
pupa.exceptions.ScrapeValueError: validation of Organization 02d08d8d-159f-11e8-8ce2-509a4c59aa96 failed:
        'Membership.asp?CommitteeID=1012&HouseID=2&SessionID=111' is not a 'uri'
```

If you call make_absolute_url on the lxml.html document that should help w/ this

**Comment by sriks123** [comment]
would 
doc = lxml.html.fromstring(data)
        member_roles = doc.xpath("//td[@id='cont']/a/following-sibling::text()[1]").make_absolute_url()
work?

**Pull request merged**